src/manual/using/integration/guidelines.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Guidelines
+order: 5
+---
+
+Before jumping in to the technical details, a few words of caution are required
+regarding integrating with Taskcluster, especially with user-provided
+credentials.
+
+## Careful with Credentials
+
+When a user clicks "Grant" for your service, they are trusting your service
+with their credentials.  For some users, those credentials are very powerful.
```

Comment:
Note: double spaces are not HTML-renderable.

Reply:
Yeah, vim adds them when I re-wrap lines.